openra/utility.py

```diff
@@ -531,11 +517,10 @@ def LintCheck(item, fullpath="", parser=settings.OPENRA_ROOT_PATH + list(reverse
                 response['error'] = True
                 response['response'] = 'can not find .oramap'
                 return response
-            fullpath = path + filename
+            fullpath = os.path.join(path, filename)
 
         os.chmod(fullpath, 0o444)
```

Comment:
It would be nice if we could remove these too... we should be mounting the map read-only, but i'm less certain that we will never modify it even despite that.

Reply:
Again as above - if could be to prevent that map being altered while it's reading it. It should be in a temporary directory with a unique name to prevent this ideally, but this is a separate issue to the PR.